openwisp_radius/api/serializers.py

```diff
@@ -359,7 +360,11 @@ def save(self):
             'from_email': getattr(settings, 'DEFAULT_FROM_EMAIL'),
             'email_template_name': ('custom_password_reset_email.html'),
             'request': request,
-            'extra_email_context': {'password_reset_url': password_reset_url},
+            'extra_email_context': {
+                'subject': _(f'Password reset on {get_current_site(request).name}'),
```

Comment:
This will not work.. f strings do not work with gettext translations, please refer to the last paragraph of this django docs section https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#standard-translation.
Please convert it as the django docs recommends and then please add the translations for this message to all the supported locales (using gtranslate).